documentation/src/docs/asciidoc/user-guide/migration-from-junit4.adoc

```diff
@@ -124,3 +124,19 @@ include::{testDir}/example/IgnoredTestsDemo.java[tags=user_guide]
 
 WARNING: JUnit 4 `@Ignore` support in JUnit Jupiter is currently an _experimental_
 feature. Consult the table in <<api-evolution-experimental-apis>> for detail.
+
+[[migrating-from-junit4-parallel-support]]
+=== Limited Parallel Execution Support
+
+As of 5.8, limited parallel execution is supported by setting the
+`junit.vintage.execution.parallel.enabled` configuration parameter to `true`. In addition, the
+following properties with the `junit.vintage.execution.parallel.config.` prefix are also supported.
+
+.Parallel Configuration Properties
+|===
+| Parameter Name | Potential Values | Description |
+| `strategy` | `fixed`, `dynamic` (default), or `custom` | If `fixed`, set an explicit number of concurrent tests to run based on the `fixed.parallelism` property. If `dynamic` or not specified, use the number of processor cores multiplied by `dynamic.factor`. If `custom` provide an implementation of `ParallelExecutionConfigurationStrategy` in `custom.class` to configure the parallelism. |
+| `fixed.parallelism` | positive integer | The maximum number of concurrent tests to run. Only applicable if `strategy=fixed`. |
+| `dynamic.factor` | positive floating point number (can be less than one) | Set the number of concurrent tests to be a multiple of the number of processor cores. Only applicable if `strategy=dynamic`. |
+| `custom.class` | Fully-qualified class name that implements `ParallelExecutionConfigurationStrategy`. | Provide a custom approach to configuring parallelism. The class must implement `org.junit.platform.engine.support.hierarchical.ParallelExecutionConfigurationStrategy`. |
+|===
```

Comment:
These are the same as for Jupiter, right? Let's move Jupiter's current section to advanced topics, generalize it and reference it from Jupiter and Vintage.

Reply:
This is done.